lib/SIL/SILVerifier.cpp

```diff
@@ -4345,6 +4345,40 @@ class SILVerifier : public SILVerifierBase<SILVerifier> {
     }
     
     SILInstructionVisitor::visitSILBasicBlock(BB);
+
+    // Visit the instructions in the BB, to find holes in the lexical scopes.
+    // We use a fairly simple heuristic here (Explain heuristic).
+    llvm::DenseSet<const SILDebugScope *> AlreadySeenScopes;
```

Comment:
(nit) I'd move this down until after the additional checks, closer to the for loop.